scripts/build/build_node/Platform/Linux/package-list.ubuntu-bionic.txt

```diff
@@ -4,6 +4,7 @@
 cmake/3.20.1-0kitware1ubuntu18.04.1     # For cmake
 clang-6.0                               # For Ninja Build System
 ninja-build                             # For the compiler and its dependencies
+java-1.8.0-amazon-corretto-jdk          # For Jenkins and Android
```

Comment:
Any reason to use v8 here instead of v11? I think we're using Corretto 11 on the Ubuntu 18 nodes.

Reply:
I'll update to Corretto 11. I think I might have that gotten mixed up somewhere